Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreter.cs

```diff
@@ -261,6 +248,14 @@ class AstPythonInterpreter : IPythonInterpreter, IModuleContext, ICanFindModuleM
             return null;
         }
 
+        public IPythonModule ImportModule(string name) {
+            var token = Debugger.IsAttached ? CancellationToken.None : new CancellationTokenSource(5000).Token;
+            var impTask = ImportModuleAsync(name, token);
+            if (!impTask.Wait(10000)) {
```

Comment:
I'd rather not time out here while debugging either, and ideally we should only have the IsAttached check in debug builds.

Reply:
Sometimes I had to debug release builds but sure :-)